src/sentry/types/activity.py

```diff
@@ -27,6 +27,7 @@ class ActivityType(Enum):
     # The user has reprocessed the group, so events may have moved to new groups
     REPROCESS = 22
     MARK_REVIEWED = 23
+    ONGOING = 24
```

Comment:
Maybe something like `AUTO_SET_ONGOING` to clarify it's an automatic transition?